File: Java/Day02Assignment/TShirt.java
package com.sonata;

public class TShirt {
	String color;
	String material;
	String design;
	TShirt(String col, String mat, String desi){
		this.color = col;
		this.material= mat;
		this.design=desi;
		}
	TShirt(){}
public void display() {
		
		System.out.println("TShirt color: "+color);
		System.out.println("TShirt material: "+material);
		System.out.println("TShirt design: "+design);
		System.out.println();
	}
	public static void main(String[] args) {
		TShirt small=new TShirt("Orange","Silk","Round-Neck");
		small.display();
		TShirt large=new TShirt("Blue","Cotton","Collored");
		large.display();
		TShirt xlarge=new TShirt("Red","Wool","Sports");
		xlarge.display();
		

	}
	
}

File: Typescripts/TShirt.ts
class TShirt{
    color:string;
    material:string;
    design:string;
    size:String;
    constructor(c:string,m:string,d:string,s:string){
        this.color=c;
        this.material=m;
        this.design=d;
        this.size=s;

    }
    display(){
        console.log("\n Color - " + this.color+" \n Material - " + this.material + "\n Design - "  + this.design + "\n Size- "  + this.size );
    }
}
var small = new TShirt("Orange","cotton","round-neck","small");
small.display();
var large = new TShirt("yellow","lenin","V-neck","Large");
large.display();
var xlarge = new TShirt("blue","silk","Collered-neck","Xtra-Large");
xlarge.display();